js/main.js

```diff
@@ -0,0 +1,81 @@
+const OBJECT_PHOTO_COUNT = 25; // кол-во объектов фото
```

Comment:
Такие комментарии лучше, потому что редактор привязывает их к переменным и они видны при ховере на них
```suggestion
/** кол-во объектов фото */
const OBJECT_PHOTO_COUNT = 25; 
```

Reply:
Ок, принято